### ISSUE TITLE
octovis bad_alloc error

### ISSUE THREAD

**Issue by alastochkinheroku** [OctoMap/octomap#266]
Hello! I was resently tried your library and I very like it. But I have some problems with viewing middle-size files (approx 9 Mb) with resolution 0.05. When I open it with 16 depth, programs crushing:
`0x76CAC54F in octovis.exe 0x0034BD58` 
and it falling in ViewerGui.cpp:295.
OS: Windows 7
Build tools: msvc2015
Qt: 4.8.7
Octomap version 1.9.1

**Comment by alastochkinheroku** [comment]
[octomap.zip](https://github.com/OctoMap/octomap/files/3886462/octomap.zip)
![bad allocation](https://user-images.githubusercontent.com/12203488/69538872-51318e00-0f94-11ea-9d3e-df099bf2e416.png)
Attached model and screen. Thanks.

**Comment by ahornung** [comment]
Sorry, I can't reproduce that. It works just fine under Linux with your provided file. Does it work for any file provided at http://ais.informatik.uni-freiburg.de/projects/datasets/octomap/ ?

Maybe it's a problem in your setup or specific to octovis under Windows.

**Comment by alastochkinheroku** [comment]
Yes, I have error also with campus file http://ais.informatik.uni-freiburg.de/projects/datasets/octomap/fr_campus.bt.gz.
Something wrong with windows build. I think we can close this ticket, I'll try to recompile or use viewer on Linux. Thank you.
